x/evm/client/cli/query.go

```diff
@@ -147,3 +149,61 @@ func GetParamsCmd() *cobra.Command {
 	flags.AddQueryFlagsToCmd(cmd)
 	return cmd
 }
+
+func GetVirtualFrontierBankContractByDenomCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "vfc-by-denom DENOM",
+		Short:   "Get the virtual frontier bank contract by denom",
+		Example: "dymd query evm vfc-by-denom <denom>",
```

Comment:
```suggestion
		Use:     "vfc-erc20-by-denom [denom]",
		Short:   "Get the virtual frontier bank contract by denom",
		Example: "dymd query evm vfc-erc20-by-denom <denom>",
```
In the future, there are more types of vfc, not just vfbc so imo we need to specify the type.